package/MDAnalysis/coordinates/PDB.py

```diff
@@ -391,14 +391,25 @@ def _read_frame(self, frame):
             elif line[:6] == 'CRYST1':
                 # does an implicit str -> float conversion
                 try:
-                    self.ts._unitcell[:] = [line[6:15], line[15:24],
-                                            line[24:33], line[33:40],
-                                            line[40:47], line[47:54]]
+                    cell_dims = np.array([line[6:15], line[15:24],
+                                  line[24:33], line[33:40],
+                                  line[40:47], line[47:54]], 
+                                  dtype=np.float32)  
                 except ValueError:
                     warnings.warn("Failed to read CRYST1 record, "
                                   "possibly invalid PDB file, got:\n{}"
                                   "".format(line))
-
+                else:
+                    if (np.array(cell_dims, dtype=np.float32) == 
+                        np.array([1, 1, 1, 90, 90, 90], 
+                        dtype=np.float32)).all():
+                        warnings.warn("1 A^3 CRYST1 record," 
```

Comment:
That probably should say "A 1^3 cell was found in the CRYST1 record"
Also "its" -> "it's" (i.e. it is), but here I would go with "this is usually a placeholder value in cryo-em structures" (the user of "for" is a bit ambiguous).

Reply:
Very true. I'm sorry for that grammar!

Reply:
No need to be happens to all of us, as you can see from my own comment having a few typos 🙃